website/cue/reference/remap/functions.cue

```diff
@@ -77,6 +77,26 @@ remap: {
 		   match criteria. We recommend using date ranges as the _only_ criteria when the enrichment
 		   data set is very small.
 
+		For `geoip` enrichment tables this condition needs to be a VRL object with a single key-value pair
+		whose value needs to be a valid IP address. Example: `{"ip": .ip }`. If a return field is expected
+		and without a value, `null` will be used. This table can return following fields:
+
+		* `autonomous_system_number`
+		* `autonomous_system_organization`
+		* `isp`
+		* `organization`
```

Comment:
These fields are returned for an isp db and the other ones are for all other types of db. Can we make that clear here?